packages/create-docusaurus/package.json

```diff
@@ -23,6 +23,7 @@
   "license": "MIT",
   "dependencies": {
     "@docusaurus/logger": "2.0.0-beta.20",
+    "@docusaurus/utils": "2.0.0-beta.20",
```

Comment:
I know it's already very bloated, but I'd like to point out that a bare install of `create-docusaurus` is 8.1M, which is a bit too much for a scaffolding utility. I'd like to lower its size gradually, like avoiding lodash and `@docusaurus/utils`

Reply:
Agree, but it will be removed after migrating to Execa so I'd rather increase this temporarily rather than creating a temporary smaller package

Reply:
Sure, I agree!